### DISCUSSION TITLE
Import maps and poi

### DISCUSSION THREAD

**Discussion by OAStubby** [devemux86/cruiser#407] [Q&A] [answered]
Is there a procedure to import, in the android version, maps and poi of OpenAndroMaps already downloaded on the PC without having to create the corresponding .txt file by hand?

**Comment by devemux86** [comment, marked as answer]
@OAStubby 

External maps can be opened with the menu "Map | External maps"
(enable the "Settings | Application | Advanced | Extended menu")

_External_ POI cannot be supported because of Android restrictions.

---

But it is more simple to use OpenAndroMaps using the download manager in Cruiser.
(it downloads automatically maps + poi)
